.circleci/config.yml

```diff
@@ -230,8 +230,8 @@ jobs:
           command: ./gradlew assembleSelfSignedRelease
 
       - run:
-          name: Check APK size isn't larger than 11.1MB (11639193 * 1024 * 1024)
-          command: if [ $(ls -l collect_app/build/outputs/apk/selfSignedRelease/*.apk | awk '{print $5}') -gt 11639193 ]; then exit 1; fi
```

Comment:
It looks like 11.1MB should be represented as 11100000 Bytes not 11639193 because I had to bump the limit to 11.8 and 11800000 is enough I don't need 12373196. So the size is in decimal not binary. I was thinking about it reviewing your pr where you have added that change but didn't have time to check it back then.